tools/cd_hit_dup/cd_hit_dup.xml

```diff
@@ -0,0 +1,135 @@
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+        <exit_code range=":-1" />
+    </stdio>
+
+    <command><![CDATA[
+        cd-hit-dup
+        -i "${ fastq_input.fastq_input1 }"
+        #if str( $fastq_input.fastq_input_selector ) == "paired":
+            -i2 "${ fastq_input.fastq_input2 }"
+        #elif str( $fastq_input.filter_chimeras.filter_chimeras_selector ) == "true":
+            -f "true"
+            -s "${ fastq_input.filter_chimeras.min_chimeric_length }"
+            -a "${ fastq_input.filter_chimeras.abundance_cutoff }"
+            -b "${ fastq_input.filter_chimeras.abundance_ratio }"
+            -p "${ fastq_input.filter_chimeras.dissimilarity_control }"
+        #end if
+        -u "${ prefix_length }"
+        -m "${ match_length }"
+        #if str( $mismatches_allowed ) != "":
+            #if float( str( $mismatches_allowed ) ) == int( float( str( $mismatches_allowed ) ) ):
+                -e "${ int( float( str( $mismatches_allowed ) ) ) }"
+            #else:
+                -e "${ mismatches_allowed }"
+            #end if
+        #end if
+        -d "${ description_length }"
+        -o "output"
+    ]]>
+    </command>
+    <inputs>
+        <conditional name="fastq_input">
+            <param name="fastq_input_selector" type="select" label="Single or Paired-end reads" help="For joined Paired-end reads choose Single.">
+                <option value="paired">Paired</option>
+                <option value="single" selected="True">Single</option>
+            </param>
+            <when value="paired">
+                <param name="fastq_input1" type="data" format="fastqsanger,fasta" label="Select first set of reads" help="Specify dataset with forward reads"/>
+                <param name="fastq_input2" type="data" format="fastqsanger,fasta" label="Select second set of reads" help="Specify dataset with reverse reads"/>
+            </when>
+            <when value="single">
+                <param name="fastq_input1" type="data" format="fastqsanger,fasta" label="Select read dataset" help="Specify dataset with single reads"/>
+                <conditional name="filter_chimeras">
+                    <param name="filter_chimeras_selector" type="select" label="Filter out chimeric clusters">
+                        <option value="true">Yes</option>
+                        <option value="false" selected="True">No</option>
+                    </param>
+                    <when value="true">
+                        <param name="min_chimeric_length" type="integer" value="30" min="20" label="Minimum length of common sequence shared between a chimeric read and each of its parents" help="-s"/>
+                        <param name="abundance_cutoff" type="integer" value="1" min="1" label="Abundance cutoff" help="-a; Tool Author recommend default of 2, but this would require the chimera itself to need 2 copies"/>
```

Comment:
Any comments/concerns for "Tool Author recommend default of 2, but this would require the chimera itself to need 2 copies", since I set the default to 1, not the program's recommended 2.